README.md

```diff
@@ -255,6 +255,12 @@ Hikari does not include a command framework by default, so you will want to pick
 - [`tanjun`](https://github.com/FasterSpeeding/Tanjun) - a flexible command framework designed to extend Hikari.
 - [`crescent`](https://github.com/magpie-dev/hikari-crescent) - a command handler for Hikari that keeps your project neat and tidy.
 
+There are also third party libraries to help you manage components:
+
+- [`yuyo`](https://github.com/FasterSpeeding/Yuyo/) - a collection of utility functions and classes designed to enhance hikari.
```

Comment:
Ngl including Yuyo here feels weird since its not a component library, it's just a general utility library.

Reply:
Ok, ill remove it